test/intl402/supportedLocalesOf-unicode-extensions-ignored.js

```diff
@@ -11,20 +11,18 @@ includes: [testIntl.js]
 ---*/
 
 testWithIntlConstructors(function (Constructor) {
-    var info = getLocaleSupportInfo(Constructor);
 
     // this test should work equally for both matching algorithms
     ["lookup", "best fit"].forEach(function (matcher) {
+        var opt = {localeMatcher: matcher};
+        var info = getLocaleSupportInfo(Constructor, opt);
         var allLocales = info.supported.concat(info.byFallback, info.unsupported);
         allLocales.forEach(function (locale) {
             var validExtension = "-u-co-phonebk-nu-latn";
             var invalidExtension = "-u-nu-invalid";
-            var supported1 = Constructor.supportedLocalesOf([locale],
-                {localeMatcher: matcher});
-            var supported2 = Constructor.supportedLocalesOf([locale + validExtension],
-                {localeMatcher: matcher});
-            var supported3 = Constructor.supportedLocalesOf([locale + invalidExtension],
-                {localeMatcher: matcher});
+            var supported1 = Constructor.supportedLocalesOf([locale], opt);
+            var supported2 = Constructor.supportedLocalesOf([locale + validExtension], opt));
+            var supported3 = Constructor.supportedLocalesOf([locale + invalidExtension], opt));
```

Comment:
This test is not syntactically correct (there is a redundant closing parenthesis on this and the previous line).

Reply:
sorry, I think it just got fixed in https://github.com/tc39/test262/commit/356afd79a81e232a3d1fe452dbe1e79ecdc17d38#diff-29fe459e743829b8576655c217425f9c83220c67d90264cfc55a959b8c2c6350